src/handlers/tracker.ts

```diff
@@ -264,7 +265,7 @@ export class TrackerHandler implements IDisposable {
 
       const code = editor.model.value.text;
       const codeId = this.debuggerService.getCodeId(code);
-      if (source !== codeId) {
+      if (widget.title.caption !== source && source !== codeId) {
```

Comment:
This assumes the `caption` is always equal to the full path.

We could think of a better way to do this by maybe adding extra data to the `editor`.

Reply:
Or we could use an `AttachedProperty`, potentially. But for now, this seems fine.